src/lib/components/svg-view/Label.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+	import DraggableSvg from "./DraggableSVG.svelte";
+	import type { iPoint } from "$lib/interfaces/iPoint";
+
+	export let parentPosition: iPoint;
+	export let position: iPoint;
+	export let text: string;
+
+	const centerOffset = 5;
+</script>
+
+<DraggableSvg bind:position>
+	<!-- line from parrent to text label, make dotted-->
+	<line
+		stroke-dasharray="5,5"
+		d="M5 20 l215 0"
+		x1={parentPosition.x}
+		y1={parentPosition.y}
+		x2={position.x}
+		y2={position.y}
+		stroke="gray"
+		stroke-width="2"
```

Comment:
As I mentioned in the original review, you need to move these variables to CSS in many of your files, not just at the exact point I have written this comment.